### ISSUE TITLE
Ingame menu for civil settlers

### ISSUE THREAD

**Issue by Ne-X-uS** [jsettlers/settlers-remake#553]
Hi,

haven't found the issue for missing ingame menu for civilian settlers. You know, the one where you can setup the ratio between bearers, builders and bulldozers.

With a normal resource set and a build row of 2 Woodcutters, 1 Forester, 1Sawmill and 1 Quarry, perhaps +2 Towers, nearly all of my free bearers run to the tool stacks and transform first into bulldozers and second into builders. That's quite uncool, because in the last game, after that rush to tools, there are only three bearers left! That's not much to transport goods to a big house I obviously needed.

**Comment by andreas-eberle** [comment]
@Ne-X-uS: Yes, at the moment there is neither a menu nor the game logic for limitting the flatteners and the builders. So thanks for opening this ticket, so we can keep track of this.

**Comment by Ne-X-uS** [comment]
I've never worked with github, otherwise I'd like to have an eye on the game logic for having at least a standard value of 40 30 30 (bearers, flatteners, builders).

Do you have a code doku where to find what?

**Comment by andreas-eberle** [comment]
Well, at the moment the game logic does not support limitting the number of flatteners and builders at all. The main problem here is that the number of settlers are not counted yet as it isn't trivial to count them per partition (connected area of a player). So you'd need to dive deep to do this....

Nevertheless, the game logic can be found here: https://github.com/jsettlers/settlers-remake/tree/master/jsettlers.logic

**Comment by Ne-X-uS** [comment]
Okay, I had a closer look at the logic.
Although I understand most of the written code, the adaptation of this code is way beyond my skill.

I think I rest as a tester and keep reporting issues/bugs.
